source/features/toggle-files-button.tsx

```diff
@@ -13,7 +13,7 @@ import observeElement from '../helpers/simplified-element-observer';
 const cacheKey = 'files-hidden';
 
 function addButton(): void {
-	const filesHeader = select('.Box-header--blue .Details > :last-child > ul');
+	const filesHeader = select('.Box-header .Details > :last-child > ul');
```

Comment:
Sadly I don't think there's much we can do about this brittle selector.
- replace `.Box-header` with `.hx_commit-tease` (it seems to be a unique class name on the relevant pages)
- use the select + closest strategy, maybe with `[title^="View all commits by"]` to reach the `.Details` directly

Reply:
`.octicon-history`?